docs/Using-EthSigner/Azure.md

```diff
@@ -31,8 +31,9 @@ option set to `8590` to avoid conflict with the default EthSigner listening port
     ```
 
 !!! important 
-    To use EthSigner for signing transactions, the Ethereum client genesis file must include at least the 
-    `byzantium` milestone.
+    EthSigner requires a [chain ID](https://besu.hyperledger.org/en/stable/Concepts/NetworkID-And-ChainID/) to be 
+    used when signing transactions. The Ethereum client genesis file must include at least the Spurious Dragon
```

Comment:
Yeah - I know this is largely a copy of the previously accepted text ... but ...

.. used when signing transactions. Thus the downstream Ethereum client's chain must be operating in a milestone which supports "replay protection" - i.e. SpuriousDragon (defined as eip158 in the genesis file) or later.

Reply:
Fixed